gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=com.exactpro.th2
-version=0.0.6
+version=0.0.7
```

Comment:
issue: Please, rollback this change. The version is automatically updated after the previous release. No need to do it again